workspace/files/ui/templates/files/ui/partials/_events_list.html

```
@@ -0,0 +1,84 @@
+          revoked a public link
+        {% else %}
+          {{ event.get_action_display }}
+        {% endif %}
+      </p>
+      <time class="text-xs text-base-content/40">{{ event.created_at|timesince }} ago</time>
+    </div>
+  </li>
+  {% endfor %}
+</ul>
+
+{% if total_event_count > events_limit %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)